Gemfile.lock

```diff
@@ -104,13 +104,7 @@ GEM
       rack-test (>= 0.6.3)
       regexp_parser (>= 1.5, < 3.0)
       xpath (~> 3.2)
-    capybara-webmock (0.6.0)
```

Comment:
weird that this downgraded 5 minor versions. I'm theoretically ok with it though since it's just for running tests and not something that would be present when deployed to a production server

Reply:
Might it be just my local environment changes? Can someone do the same webdriver upgrade and see if `Gemfile.lock` get the same changes as I do?